src/theme/git_symbol.rs

```diff
@@ -0,0 +1,52 @@
+use serde::Deserialize;
+
+#[derive(Debug, Deserialize, PartialEq, Eq)]
+#[serde(rename_all = "kebab-case")]
+#[serde(deny_unknown_fields)]
+#[serde(default)]
+pub struct GitSymbolTheme {
+    pub default: String,
+    pub unmodified: String,
+    pub new_in_index: String,
+    pub new_in_workdir: String,
+    pub deleted: String,
+    pub modified: String,
+    pub renamed: String,
+    pub ignored: String,
+    pub typechange: String,
+    pub conflicted: String,
+}
+
+impl Default for GitSymbolTheme {
+    fn default() -> GitSymbolTheme {
+        GitSymbolTheme {
+            default: "_".into(),
+            unmodified: "_".into(), // "\u{f00c}
+            new_in_index: "\u{f067}".into(),
+            new_in_workdir: "?".into(),
+            deleted: "\u{f014}".into(), // or f06
+            modified: "\u{f8ea}".into(),
+            renamed: "\u{f8ea}".into(),
+            ignored: "!".into(),
```

Comment:
I would prefer a less eye catch symbol for `ignore`, like `I` in `exa`

Reply:
Done.